src/main/java/com/dataguise/saas/service/JWTService.java

```diff
@@ -0,0 +1,56 @@
+package com.dataguise.saas.service;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.apache.log4j.Logger;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Date;
+
+import static java.util.Collections.emptyList;
+
+public class JWTService {
```

Comment:
**Reporter**: PMD    
    **Rule**: ClassNamingConventions
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/service/JWTService.java L16
    
    The utility class name &#39;JWTService&#39; doesn&#39;t match &#39;[A-Z][a-zA-Z0-9]+(Utils?|Helper)&#39;&#10;&#10;&#10;Code Style https://pmd.github.io/pmd/pmd_rules_java_codestyle.html#classnamingconventions
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1935121984>*